classes/site/Version.inc.php

```diff
@@ -13,7 +13,8 @@
  *
  * @brief Describes system version history.
  */
-
+use Composer\Semver\VersionParser;
+use Composer\Semver\Semver as Sem;
```

Comment:
Was this aliased because there was another `Semver` reference already in scope, or for another reason?

Reply:
It was aliased because it wasn't working without the alias originally. It now works without the alias, so I removed the alias.